Kalyani/HTML/Assginment3/KalyaniA3_2contact.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Qust2</title>
+    <link rel="icon" type="image/x-icon" href="./assets/favicon.ico" />
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+<style>
+  body {
+        background-color: #3fe1f7;
+        color: #6fa6a0;
+      }
+      a:link {
+        color:red;
```

Comment:
there should be always space after colon, do this change everywhere

Reply:
mam i have done all changes